utilities/common.py

```diff
@@ -291,7 +291,7 @@ def __call__(self, parser, namespace, values, option_string=None):
     parser.add_argument("-p", "--postfix", type=str, help="Postfix for output file name", default=None)
     parser.add_argument("--forceDefaultName", action='store_true', help="Don't modify the name of the output file with some default strings")
     parser.add_argument("--theoryCorr", nargs="*", type=str, action=NoneFilterAction,
-        default=["scetlib_dyturbo", ], choices=theory_corrections.valid_theory_corrections(), 
+        default=["scetlib_dyturbo", "scetlib_dyturboCT18ZVars", "scetlib_dyturboCT18Z_pdfas"], choices=theory_corrections.valid_theory_corrections(), 
```

Comment:
This means that the scetlib-dyturbo CT18Z pdf variations are loaded (but not necessarily used in the final fit?)

Reply:
Yeah, wasn't sure if I wanted to make that default or not. But probably it doesn't hurt much?